engine/link/link.asm

```diff
@@ -83,23 +83,23 @@ endc
 	ld hl, wLinkBattleRNPreamble
 	ld de, wEnemyMon
 	ld bc, SERIAL_RN_PREAMBLE_LENGTH + SERIAL_RNS_LENGTH
-	vc_hook Network358
+	vc_hook ExchangeBytes1
```

Comment:
```suggestion
	vc_hook wireless_ExchangeBytes_1
```
(likewise for the rest; I think that makes it clear that the hook is for changing serial to Wi-Fi communication.)